.circleci/config.yml

```diff
@@ -48,6 +50,53 @@ commands:
           paths:
             - ~/.cache/yarn
 
+  # Custom version of "checkout" that checks out a SHA deployed to DEPLOY_URL env
+  # Adapter from https://gist.github.com/drazisil/e97dc21454120251472154de1c1b1c7b
+  checkout-deployed-sha:
+    parameters:
+      sha_url:
+        type: string
+    steps:
+      - run:
+          name: Custom Checkout of Deployed SHA
+          command: |
+            DEPLOYED_SHA=$(curl --silent << parameters.sha_url >> | jq -r '.git_sha')
```

Comment:
If we ever get a `stages/int` branch that we can rely on....this could all go away and we could let CircleCI do this

Reply:
In the mean time, this is delightfully clever, but not to the point of being incomprehensible.